test/numbers.jl

```diff
@@ -1,5 +1,7 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
+≣(a, b) = isequal(a, b) # convenient for comparing NaNs
```

Comment:
As a general style note, when we define operators to be equivalent to functions, we generally use `const ≣ = isequal`.  This has some advantages: docstrings for `≣` work, typing `≣` at the REPL prints `isequal` so it is elf-documenting to some extent, and compilation time is decreased.  Not that these considerations matter much in a test file, however.